src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMNavigator.java

```diff
@@ -145,6 +145,14 @@
     @Restricted(NoExternalUse.class)
     @RestrictedSince("2.2.0")
     private transient String bitbucketServerUrl;
+    @Deprecated
+    @Restricted(NoExternalUse.class)
+    @RestrictedSince("2.2.0")
+    private transient String bitbucketJenkinsRootUrl;
```

Comment:
Why two transient values? and why are they both marked deprecated?

Reply:
Copypasted from other fields that worked. Otherwise, sorry but no idea what this really means (not a Java dev here) :-)